src/CMakeLists.txt

```diff
@@ -13,6 +13,7 @@ add_subdirectory (cpp)
 endif()
 
 add_subdirectory (clang-c-frontend)
+add_subdirectory (solidity-frontend)
```

Comment:
Could you add this as an option? https://github.com/esbmc/esbmc/blob/master/scripts/cmake/Options.cmake#L24

Reply:
ACK

Reply:
Done